.github/workflows/coverity.yaml

```diff
@@ -0,0 +1,48 @@
+name: Coverity
+on:
+#  schedule:
+#    # run daily 20:00 on master branch
+#    - cron: '0 20 * * *'
+  push:
+#    branches:
+#      - coverity_scan
+jobs:
+  coverity:
+    name: Coverity
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        pg: [11]
```

Comment:
I believe the Coverity Scan needs to run on both PG 11 and PG 12.

Reply:
it will run on both currently debugging some 11 issues

Reply:
👍 Thank you for clarifying!